src/components/Layout.js

```diff
@@ -6,12 +6,9 @@ import {
 import { bindActionCreators } from 'redux';
 import { Breadcrumb } from 'react-breadcrumbs';
 import { connect } from 'react-redux';
-//<<<<<<< HEAD:src/components/Layout.js
```

Comment:
oops sorry. my bad!

Reply:
No worries, this should've got caught during review